Detectors/EMCAL/base/include/EMCALBase/Geometry.h

```diff
@@ -34,6 +32,9 @@ namespace EMCAL
 {
 class ShishKebabTrd1Module;
 
+using doublevect = std::vector<double>;
```

Comment:
typenames should follow the type convention `DoubleVect`. This will be enforced soon.

Reply:
Fixed.

Reply:
You should not use "using" in headers. Moreover it is considered bad practice to have such a generic aliases, as they confuse the reader. You read code far more times you write it, so you should not worry about saving a few characters (of course for a specific type it's a different matter).

Reply:
I completely agree with you to ban using directives from header files in case they are used for removing namespaces as this is a super dangerous procedure just in order to save a few characters, which is a wide-spread approach in AliRoot/AliPhysics code. Here however the using directive is used as a typedef, which is even declare within the EMCAL namespace. Furthermore I don't really see where it has any impact on readability: The name "DoubleVect" should be self-explaining.

Reply:
I agree somewhat with @ktf that "std::vector<double>" is a bit clearer than DoubleVect and the gain in typing is in any case minor.

Reply:
Not sure if `doublevect` is so clear, can be `vector` of type `double`, or a somehow duplicated vector, some vector which is there twice. My first association goes more in the second direction, and that makes a simple thing confusing

Reply:
`DoubleVect` serves no purpose but to save a couple of keystrokes and hide the actual type, possibly generating confusion. If you really need a typedef, you should give it some specific type meaning like "WeightVector", "EnergyVector" or similar. In any case do not keep it at namespace level, but hide it inside the class itself. Having it at namespace level means that everyone, in any namespace will have to agree on what your type is, which is bad.

Reply:
OK, I will change this back to std::vector in the commit together with the std::set issue.